Fix `cannot find module db error` in JavaScript template (#3940)
blitz-js/blitz: .changeset/light-squids-draw.md

---
"blitz": patch
"@blitzjs/generator": patch
---

> Fix `cannot find module db error` in JavaScript template. Replaced requiring the config using `esbuild` instead paring the AST to get the `cliConfig`. Allow added option 
> to find the `blitz-server` file in `src` directory for this logic.

Author: ```suggestion
Fix `cannot find module db error` in JavaScript template. Replace requiring the config using `esbuild` with parsing using `jscodeshift` to get the `cliConfig` values. Added logic to find the `blitz-server` file in `src` directory 
```
